### ISSUE TITLE
[Question] DYN_TYPES_Error: The container is full

### ISSUE THREAD

**Issue by kteriusghub** [eProsima/SOSS-DDS#51]
I built with 'feature/xtypes-dds' of SOSS-DDS, and gave 100 elements to each of two vector members through the 100-times push_back callings, and used the TCP transfer between the client and the server of SOSS. The result was good to send and receive the following customized ROS2 topic message type (as the LiDAR data)

   float ranges[];
   float intensities[];

But, when I gave the 200-times push_back callings to the both of vector member, the client side of SOSS showed the following message. And the message codes are found from DynamicData.cpp of Fast-RTPS.
 "[DYN_TYPES Error] Error inserting data. The container is full. -> Function insert_sequence_data"

As we know, some LiDARs generate more than a thousand intensitie data, so these vector members need to have more than 1000 elements. 
I want to know if it is the known issue of SOSS-DDS, or only my issue. If it is the know issue, was it fixed?

**Comment by richiware** [comment]
Are you defining your type in IDL format? In this case, can you copy it here?

Thanks.

**Comment by kteriusghub** [comment]
@richiware 
No, I didn't define it on both of SOSS and SOSS-DDS, however, I included the ROS2 package name with the structure type to CMakeLists.txt, using soss_rosidl_mix( PACKAGES ...).
And then I mentioned 'types-from: ros2' to the yaml file without IDL description.
To prevent the overflow of dynamic data container, should I define the IDL to the yaml file and configure the remap?

**Comment by richiware** [comment]
Could you provide the `msg` file which causes the error?

**Comment by kteriusghub** [comment]
@richiware 
This is the msg file.
 --- std_msgs/Header header
 --- float32[] ranges
 --- float32[] intensities

And, I filled the vector elements and publish data as this way.
--- auto msg = LidarData();
--- for (int i = 0; i < 200; i++) {
------- msg.ranges.push_back(0.1);
------- msg.intensities.push_back(0.5);
------- }
--- LidarDataPublisher->publish(msg);
